source/renderer/zettlr-renderer.js

```diff
@@ -120,7 +121,7 @@ class ZettlrRenderer {
       // Apply the custom CSS stylesheet to the head element
       global.ipc.send('get-custom-css-path', {}, (ret) => {
         let lnk = $('<link>').attr('rel', 'stylesheet')
-        lnk.attr('href', 'file://' + ret + '?' + Date.now())
+        lnk.attr('href', 'safe-file://' + ret)
```

Comment:
See above my question why the cache-breaker is apparently not needed anymore, and why we need `safe-file`, as this was never needed beforehand.

Reply:
See above: caching is now handled via headers for the safe-file protocol